packages/public/server/src/module/Api/src/Controller/ApiController.php

```diff
@@ -62,6 +65,28 @@ public function aliasAction()
         $alias = $this->params('alias');
         $instance = $this->getInstanceManager()->getInstanceFromRequest();
 
+        $userProfilePrefix = 'user/profile/';
+        if (strpos($alias, $userProfilePrefix, 0) === 0) {
```

Comment:
`0` seems to be the default for the third argument, so I'd remove it. Furthermore: I'd probably prefer to extract that into a method `startsWith` (or similar) since I wasn't aware of `strpos`.

Reply:
Also: a regex might be more readable (especially since you extract the username afterwards anyways). Would make it pretty clear that you are looking for a string 'user/profile/:username'.

Reply:
Thx, I merge it and I'll do a follow up PR...